dashboard/src/components/charts/ChartTooltip.tsx

```diff
@@ -61,18 +64,20 @@ export function ChartTooltip({
           </span>
         </div>
         {hasComparison && comparisonData && (
-          <div className='text-muted-foreground text-xs'>vs {labelFormatter(comparisonData.compareDate)}</div>
+          <div className='text-muted-foreground text-xs'>
+            {dictionary.charts.tooltip.vs} {labelFormatter(comparisonData.compareDate)}
+          </div>
         )}
       </div>
 
       <div className='mb-3'>
-        <div className='text-muted-foreground mb-1 text-xs'>Current Period</div>
+        <div className='text-muted-foreground mb-1 text-xs'>{dictionary.charts.tooltip.currentPeriod}</div>
```

Comment:
Question: Why is it `dictionary.charts.tooltip.currentPeriod` here instead of `dictionary.t('charts.tooltip.currentPeriod')`?